web/source/kmwuitoggle.js

```diff
@@ -52,9 +52,20 @@ try {
   ui.doFocus = function(someElement,focusing)
   {
     // This callback must be ignored until UI is initialized, or for touch devices (which can never initialize the UI)
-    if(!ui.initialized) return;
+    if(!ui.initialized) {
+      return;
+    }
     
-    if(window.event) someElement=window.event.srcElement;
+    if(window.event) {
+      someElement=window.event.srcElement;
+    }
+
+    // Hides the Toggle interface if a control has KeymanWeb disabled by API calls.
+    if(keymanweb._ActiveControl != null){
+      if(!keymanweb._ActiveControl.LEnabled) {
```

Comment:
Here, use the `activeControl` property passed through per previous comment thusly:

````
if(activeControl['LEnabled'])
````

This fixes the Closure Compiler's complaint. kmwuifloat.js also references this property. Ideally, in the future, we should try and eliminate these types of dependencies, but I think that's a bridge too far for this PR (and is more about code quality than a bug).